snaps/reference/snaps-api.md

```diff
@@ -864,7 +864,9 @@ await snap.request({
 ## `snap_notify`
 
 Displays a [notification](../features/notifications.md) in MetaMask or natively in the OS.
-Snaps can trigger a short notification text for actionable or time sensitive information.
+Snaps can trigger a short notification text for actionable or time sensitive information. 
```

Comment:
Existing issue, but "short" is somewhat subjective. Are we able to clarify this? For example, "Snaps can trigger a notification text of up to xx characters for actionable or time-sensitive information."

Reply:
Updated to state explicitly up to 80 characters.